src/gui/transferlistmodel.cpp

```diff
@@ -124,8 +124,8 @@ QVariant TransferListModel::headerData(int section, Qt::Orientation orientation,
             case TR_SEEN_COMPLETE_DATE: return tr("Last Seen Complete", "Indicates the time when the torrent was last seen complete/whole");
             case TR_LAST_ACTIVITY: return tr("Last Activity", "Time passed since a chunk was downloaded/uploaded");
             case TR_TOTAL_SIZE: return tr("Total Size", "i.e. Size including unwanted data");
-            default:
-                return {};
+            case TR_AVAILABILITY: return tr("Availability");
```

Comment:
We should include a description like the other columns. “Availability” isn’t necessarily an intuitive concept.

Reply:
I copied some description from libtorrent doc. Also this isn't really critical as it is meant only for translators.